test/Crossroads.Test/Utility/IgnoreOnLinuxFactAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Crossroads.Test.Utility
+{
+    public class IgnoreOnLinuxFactAttribute : FactAttribute
```

Comment:
The name is misleading. It says ignore on Linux. What it actually does is ignore when not on Windows. Suggestion: make this a bit more general. Rename to `PlatformRestrictedFactAttribute` and allow setting the supported platforms like this:
```cs
[PlatformRestrictedFact(Windows = true, OSX = true)]
```
Add one `bool` property for each member of [`OSPlatform`](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.osplatform) - since the `OSPlatform` predefined values are static fields and so they can't be used in attribute constructors.

Reply:
Thanks for the feedback. Sounds like an informative idea. I will update the pr with this.